### ISSUE TITLE
Carthage won't build - incorrect Swift version

### ISSUE THREAD

**Issue by johnrogers** [sendbird/sendbird-uikit-ios#10]
Hello there,

Here's a dump from `carthage build`. The SDK is built using the correct version of Swift, and I am able to import it, but I am unable to use UIKit:

```
 /D/***  ╍╍ beta  carthage build sendbird-uikit-ios --platform iOS                                                                                                                                              (955ms)  
*** xcodebuild output can be found in /var/folders/bf/dfxzvd0d3b71fzwt718p8tqc0000gn/T/carthage-xcodebuild.0jz0CG.log
*** Downloading sendbird-uikit-ios.framework binary at "v1.0.5"
***  Skipped installing sendbird-uikit-ios.framework binary due to the error:
	"Incompatible Swift version - framework was built with 5.1.3 (swiftlang-1100.0.282.1 clang-1100.0.33.15) and the local version is 5.2.2 (swiftlang-1103.0.32.6 clang-1103.0.32.51)."

    Falling back to building from the source
*** Skipped building sendbird-uikit-ios due to the error:
Dependency "sendbird-uikit-ios" has no shared framework schemes for any of the platforms: iOS

If you believe this to be an error, please file an issue with the maintainers at https://github.com/sendbird/sendbird-uikit-ios/issues/new
```

As a result, I can't use the framework in my project. 

Could you please update the framework, building with Swift 5.2.x? It seems like the SDK has already done so otherwise I would be running into the same issue with that.

Carthage version 0.34.0
Xcode 11.4.1

**UPDATE**

This appears to be an issue with Carthage. [See here](https://github.com/Carthage/Carthage/issues/2887).

Copying the framework into the project manually has resolved the issue for now - I don't think there's anything you guys need to do. I'll leave this here just so you're aware of it, though.

**Comment by jasoncollins-sendbird** [comment]
Hey @johnrogers I am working with Bernie and Louis on your evaluation. I'll get the engineering team to look into this.  Thanks for reporting!

**Comment by tezpark** [comment]
Hello, @johnrogers.
Thanks to your report, we were able to identify the Cartage problem in the upper version. 👍 
I will fix the problem quickly and let you know again!

**Comment by tezpark** [comment]
Hi, @johnrogers .
We have released the issue you reported in the 1.0.6 version.
Thanks for reporting 👍
